build-logic/testing/src/main/kotlin/convention.unit-testing.gradle.kts

```diff
@@ -59,12 +57,12 @@ if (project.name != "gradle-kotlin-dsl-accessors") {
 
     plugins.withType<KotlinBasePluginWrapper> {
         dependencies {
-            testImplementation(libs.junitJupiterApi)
-            testImplementation(libs.truth)
+            add("testImplementation", libs.junitJupiterApi)
```

Comment:
What is the difference?

Reply:
these acessors (testImplementation), was imported from wrong file, and in fact are not accessible here, because needed plugin (kotlin) not available during convention plugin resolution